apps/music/js/ui/views/mode_manager.js

```diff
@@ -161,7 +161,14 @@ var ModeManager = {
                        'search-from-tiles-mode', 'search-from-list-mode',
                        'picker-mode'];
```

Comment:
It might be sufficient to simply remove `'picker-mode'` from this list. I'll let Dominic comment in more detail though, since he knows more about the picker.